src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/Configs.kt

```diff
@@ -62,6 +64,7 @@ data class MarketsConfigs(
             subscribeToOrderbook = true,
             subscribeToTrades = true,
             subscribeToCandles = false,
+            retrieveSevenDaySparkline = false,
```

Comment:
should we set this to true so we can assume web will use the isNew field?

Reply:
If it's set to `true`, the Abacus would do the fetch.  It's better to turn it `true` after the web fetch is turned off, so that we don't fetch the same thing twice.